apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -105,6 +108,11 @@ public Patient read(@IdParam IdType patientId, RequestDetails requestDetails) {
 
     List<String> includeIdentifiersValues = returnIncludeIdentifiersValues(requestDetails);
 
+    Operation operation = new Operation(Operation.Endpoint.V1_PATIENT);
+    operation.setOption("by", "id");
+    operation.setOption("IncludeIdentifiers", includeIdentifiersValues.toString());
```

Comment:
These look like they're custom versions of the query params and headers sent to the system. It looks like these are also tracked in `RequestResponseLoggingFilter`.

Reply:
Correct, but separated out to make NDJSON searches/filtering easier.